src/apprt/gtk-ng/class/command_palette.zig

```diff
@@ -216,6 +217,10 @@ pub const CommandPalette = extern struct {
     /// Helper function to send a signal containing the action that should be
     /// performed.
     fn activated(self: *CommandPalette, pos: c_uint) void {
+        // add a reference to keep ourselves around until we're done
+        _ = self.ref();
+        defer self.unref();
```

Comment:
We should be able to extract the `cmd` from the list model before calling `close` (but also before signal still) so we don't have to do this, because our `cmd` object is still referenced.

Reply:
Fixed.